circle.yml

```diff
@@ -476,11 +497,14 @@ jobs:
       - run-firebase-instrumentation:
           module_target: "examples"
           variant: "debug"
+      - run-firebase-robo:
+          module_target: "examples"
+          variant: "debug"
```

Comment:
Wondering if we should move this to a separate job so it runs in parallel and we could scratch some CI / building time. What do you think?

Reply:
sure, makes sense :+1: